source/_integrations/sun.markdown

```diff
@@ -7,6 +7,7 @@ ha_release: pre 0.7
 ha_quality_scale: internal
 ha_codeowners:
   - '@Swamp-Ig'
+ha_iot_class: Assumed State
```

Comment:
```suggestion
ha_iot_class: Calculated
```

We've added an IoT class for these types of things: `Calculated`, as the state isn't from a device or service, but calculated locally.